### ISSUE TITLE
Is there a backstory behind `minLength` not passing through to `<input>`?

### ISSUE THREAD

**Issue by tony** [nkbt/react-debounce-input#155]
Is it by design that `minLength` doesn't pass through to `<input>`?

TypeScript: [lib.dom.ts for `HTMLInputElement`](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.htmlinputelement.html#minlength)
HTML: [`minlength` for `HTMLInputElement`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/minlength) on MDN

My supposition is that in React `minLength` (react) -> `minlength` (HTML)

Sandbox: https://codesandbox.io/s/react-18-playground-react-debounce-input-case-8pln0b?file=/src/index.tsx

**Comment by nkbt** [comment]
Yeah minLength is used internally because I tried to make component a drop in replacement. 

The main issue is that underlying component does not have to be an input. Could be a textarea or anything else. And it does not support minLength attribute and will throw warning about it.

**Comment by tony** [comment]
@nkbt Suggestion for what we do next?  Should we consider this closed as a wontfix since the underlying component is unknown? Is there an approach for a PR potentially?
